src/main/java/org/infernalstudios/mythostheon/MythosTheon.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Infernal Studios
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.infernalstudios.mythostheon;
+
+import net.minecraft.world.item.CreativeModeTab;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.fml.ModLoadingContext;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Mod(MythosTheon.MOD_ID)
+public class MythosTheon {
+    public static final Logger LOGGER = LogManager.getLogger();
+    public static final String MOD_ID = "mythostheon";
+
+    public MythosTheon() {
+        final ModLoadingContext modLoadingContext = ModLoadingContext.get();
+        final IEventBus modEventBus = FMLJavaModLoadingContext.get().getModEventBus();
+
+        modEventBus.addListener(this::commonSetup);
+        modEventBus.addListener(this::clientSetup);
+    }
+
+    private void commonSetup(final FMLCommonSetupEvent event) {
+    }
+
+    private void clientSetup(final FMLClientSetupEvent event) {
+    }
+
+    public static final CreativeModeTab TAB = new CreativeModeTab("MythosTheonTab") {
```

Comment:
```suggestion
    public static final CreativeModeTab TAB = new CreativeModeTab("mythos_theon_tab") {
```
should be lower_snake_case, to fit with every other identifier in the game (and in the lang file)

i'm aware that's how IE does it, but i cba to fix it there.